README.rst

```diff
@@ -111,7 +111,7 @@ Active contributors
 
 * `Phillip Chlap`_
     * `University of New South Wales`_, Australia
-    * `South Western Sydney Local Health District`_, Australia
+    * `Ingham Institute`_, Australia
```

Comment:
Logos are welcome for anyone who's company/organisation wants them. I understand if not though.

Reply:
Yeah UNSW are a bit funny about using their logo without explicit approval. But I'll keep it in mind and raise it next time I am chatting with someone who might know more around the permissions of using it.

Reply:
I was actually more thinking Ingham Institute, if they're interested...

Reply:
Ah yep, I'll talk to them about it :)